src/System.Diagnostics.Process/src/System/Diagnostics/AsyncStreamReader.cs

```diff
@@ -13,211 +13,147 @@
 
 using System.Collections.Generic;
 using System.IO;
+using System.Runtime.ExceptionServices;
 using System.Text;
 using System.Threading;
 using System.Threading.Tasks;
 
 namespace System.Diagnostics
 {
-    internal sealed class AsyncStreamReader : IDisposable
+    internal sealed class AsyncStreamReader
     {
-        internal const int DefaultBufferSize = 1024;  // Byte buffer size
+        private const int DefaultBufferSize = 1024;  // Byte buffer size
```

Comment:
Out of curiosity, and the PR author likely isn't the best person to answer this, why is this buffer so incredibly tiny? Given that a page is generally 4096 bytes, shouldn't it at least be a page in size.

Given [TN I/O Request Size](https://technet.microsoft.com/en-us/library/cc938632.aspx) article, the ideal size for reading/writing to IO is generally 64 KB. Wouldn't increasing the size of this buffer improve performance in general?

Am I missing something? (likely)